guides/common/modules/con_content-flow-in-project.adoc

```diff
@@ -38,7 +38,7 @@ image::common/system-architecture-satellite.png[Content flow in {ProjectName}]
 endif::[]
 
 .Additional resources
-* See xref:Major-{Project}-Components_{context}[] for details.
+* See xref:common/modules/con_major-project-components.adoc#Major-{Project}-Components_{context}[] for details.
```

Comment:
```suggestion
* See xref:common/modules/con_major-project-components.adoc[] for details.
```

Can we drop the anchor because it's at the top of the module?

Reply:
Probably? @apinnick Can we?

Reply:
Only one way to find out. :smile: I'll push a new commit. IIRC, it is required but does not hurt to test.